Makefile

```diff
@@ -17,6 +17,7 @@ RELEASE_DIR=_output/release
 REPO_PATH=volcano.sh/volcano
 IMAGE_PREFIX=volcanosh
 CRD_OPTIONS ?= "crd:crdVersions=v1,generateEmbeddedObjectMeta=true"
+CRD_OPTIONS_EXCLUDE_DESCRIPTION=${CRD_OPTIONS}",maxDescLen=0"
```

Comment:
Is there any effect after removing the description?

Reply:
description comes from the comment of CRD'field, and removing it has no effect to job's CRD definition, the only effect is that user use `kubectl get crd xxx -oyaml` cannot see the descrption of every field and then can check code or doc to get detail description of every field.